modules/golang/golang

```diff
@@ -16,6 +16,10 @@ go () {
 			cd `go list -f '{{.Dir}}' ".../$1" | head -n 1`
 		fi
 	;;
+	"get")
+		shift
+		command go get $(echo $1 | sed 's/^http.*:\/*//')
```

Comment:
Maybe that was the option I was missing.  When I tried `https?` on a Mac it would not match.